lib/element.js

```diff
@@ -130,7 +130,7 @@ ElementArrayFinder.prototype.all = function(locator) {
   var getWebElements = function() {
     if (self.getWebElements === null) {
       // This is the first time we are looking for an element
-      return ptor.waitForAngular().then(function() {
+      return ptor.waitForAngular(locator).then(function() {
```

Comment:
`opt_description` is supposed to be a string.  Maybe instead it should be `waitForAngular('Locator: ' + locator)`?